src/routes/docs/widget/full-api/+page.md

```diff
@@ -31,7 +31,7 @@ const widget = new Widget({
       realmPath: 'alpha', // OPTIONAL; Uses 'alpha', if not provided
       redirectUri: window.location.href, // OPTIONAL; falls back to `window.location.href`
       scope: 'openid email', // OPTIONAL; falls back to minimal 'openid email'
-      tree: 'Login', // OPTIONAL; falls back to default Login journey provided in ForgeRock
+      tree: 'Login', // NOT RECOMMENDED; OPTIONAL; falls back to default Login journey provided in ForgeRock
```

Comment:
We might want to start changing how we approach setting the `tree` value on the base config.